packages/wp-now/src/download.ts

```diff
@@ -37,15 +31,11 @@ function httpsGet(url: string, callback: Function) {
 	https.get(url, { agent }, callback);
 }
 
-function getWordPressVersionUrl(version = DEFAULT_WORDPRESS_VERSION) {
-	if (!isValidWordPressVersion(version)) {
-		throw new Error(
-			'Unrecognized WordPress version. Please use "latest", "trunk", "nightly", or numeric versions such as "6.2", "6.0.1", "6.2-beta1", or "6.2-RC1"'
-		);
-	}
```

Comment:
I've found this check is already on:
https://github.com/zaguiini/playground-tools/blob/f8e1c1e15767ae0e391317afffbf61964e7fad7b/packages/wp-now/src/config.ts#L146-L150